lib/auth/webauthn/register.go

```diff
@@ -228,20 +228,34 @@ func upsertOrGetWebID(ctx context.Context, user string, identity RegistrationIde
 	return wla.UserID, nil
 }
 
+// RegisterRequest represents fields needed to register a new webautn device.
+type RegistrationRequest struct {
+	// User is the user of the new device.
+	User string
+	// DeviceName is the name for the new device.
+	DeviceName string
+	// CreationResp is the response from the new device.
+	CreationResp *CredentialCreationResponse
```

Comment:
nit: spell out abbreviations in public APIs.

```suggestion
	// CreationResponse is the response from the new device.
	CreationResponse *CredentialCreationResponse
```

Reply:
That said, I'd be OK with `CCR` for the field name too.  🤷‍♂️